exonum/src/blockchain/transaction.rs

```diff
@@ -238,8 +252,8 @@ impl TransactionError {
     }
 
     /// Returns an optional error description.
-    pub fn description(&self) -> &Option<String> {
-        &self.description
+    pub fn description(&self) -> Option<&str> {
+        self.description.as_ref().map(|d| d.as_ref())
```

Comment:
Would it make sense to use `.map(String::as_ref)` instead of `.map(|d| d.as_ref())`? I think clippy had suggested the first version previously, but it seems it has stopped to.